server/spec/services/rpc/container_handler_spec.rb

```diff
@@ -107,23 +107,148 @@
       #puts bm
       total_time = Time.now.to_f - start_time
       expect(grid.container_logs.count).to eq(1_000)
-      expect(total_time < 0.5).to be_truthy
+      expect(total_time).to be < 0.5
     end
   end
 
-  describe '#on_container_health' do
-    it 'updates health status' do
+  describe '#save' do
```

Comment:
Yes good call I think there is some duplication going on in this spec file.  The merge got pretty nasty....